docs/tutorials/cluster.md

```diff
@@ -337,11 +337,11 @@ MiddleManager (Single-server)
 ```
 druid.worker.capacity=8
 druid.indexer.fork.property.druid.processing.numMergeBuffers=2
-druid.indexer.fork.property.druid.processing.buffer.sizeBytes=100000000
+druid.indexer.fork.property.druid.processing.buffer.sizeBytes=100MiB
```

Comment:
As in previous comment. Might be better to keep in the original form, just to match the config files as shipped to avoid confusion.

Reply:
The configuration file uses the human-readable form `100MiB` already

https://github.com/apache/druid/blob/af2ab98574582099557932e6a6370bf8dcdf3fab/examples/conf/druid/single-server/small/middleManager/runtime.properties#L35